### PR TITLE
Change base image format and PRODUCT_PATTERN_PREFIX, to match regex in push-to-obs

### PR DESCRIPTION
**Pull request by mbussolotto**
## What does this PR change?

Change base image format and PRODUCT_PATTERN_PREFIX, to match regex in push-to-obs

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed

- [x] **DONE**

## Test coverage
- No tests
- [x] **DONE**

## Links

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [x] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by deneb-alpha — APPROVED** [review]
ah, good  point... thanks

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
changelog is missing

**Comment by mbussolotto** [comment]
@deneb-alpha I added a new commit also with `PRODUCT_PATTERN_PREFIX` fix. About the changelog, I think it's not required (not completely sure...anyway it should not create any problem, cause the images are always pushed to Build Service if there are changes and we didn't release anything yet). I ticked `No changelog needed` .

File: rel-eng/push-packages-to-obs.sh [hunk 1]
```diff
@@ -272,6 +272,7 @@ while read PKG_NAME; do
           # SUSE Manager settings
           VERSION=$(sed 's/^\([0-9]\+\.[0-9]\+\).*$/\1/' ${BASE_DIR}/packages/uyuni-base)
           sed "s/^ARG INIT_BASE=.*$/ARG INIT_BASE=bci\/bci-init:15.4/" -i $SRPM_PKG_DIR/Dockerfile
+          sed "s/^ARG PRODUCT_PATTERN_PREFIX=.*$/ARG PRODUCT_PATTERN_PREFIX=patterns-suma/" -i $SRPM_PKG_DIR/Dockerfile
```
Comment (deneb-alpha) [inline rel-eng/push-packages-to-obs.sh:275]: here  we have INIT_BASE, but in the Dockerfile you added INIT_IMAGE for  server-image.  are  you sure?

File: containers/server-image/Dockerfile [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # SPDX-License-Identifier: MIT
 #!BuildTag: uyuni/server:latest
 
-ARG INIT_BASE=uyuni/init:latest
-FROM $INIT_BASE
+ARG INIT_IMAGE=uyuni/init:latest
+FROM $INIT_IMAGE
```
Comment (deneb-alpha) [inline containers/server-image/Dockerfile:5]: Why INIT_IMAGE  instead of INIT_BASE?

**Review by deneb-alpha — COMMENTED** [review]
sorry, not sure  anymore

**Review by deneb-alpha — APPROVED** [review]

**Pull request merged**